### PR TITLE
MWSS gross pay piping

### PR DESCRIPTION
**Pull request by jonnyshaw89**
### What is the context of this PR?
Added the piping for gross pay into next question

### How to review 
Enter a gross pay into each section and see that appropriate value is piped into the question text

### CODE REVIEW

File: app/data/mwss_4.json [hunk 1]
```diff
@@ -242,7 +243,7 @@
                                     "description": "",
                                     "id": "weekly-pay-question-6",
                                     "number": "2.3",
-                                    "title": "Of the [TOTAL gross PAY] paid to <em>weekly paid</em> employees in the <em>last week of December 2016</em>, what amount was:",
+                                    "title": "Of the {{answers.weekly_total_gross}} paid to <em>weekly paid</em> employees in the <em>last week of December 2016</em>, what amount was:",
```
Comment (ajmaddaford) [inline app/data/mwss_4.json]: I think there should be a pound sign in this text, it currently looks like this:
image

Same for all other occurrences.

**Review by ajmaddaford — CHANGES_REQUESTED** [review]

File: app/data/mwss_4.json (hunk 1 above)
Comment (jonnyshaw89) [reply app/data/mwss_4.json]: @ajmaddaford i have updated this.

**Review by ajmaddaford — APPROVED** [review]

**Comment by ajmaddaford** [comment]
I've added currency formatting for the piped gross pay values and pushed the rebased changes

**Review by LJBabbage — CHANGES_REQUESTED** [review]
ignore

**Review by LJBabbage — APPROVED** [review]

**Pull request merged**